core/src/main/java/com/linecorp/armeria/server/Server.java

```diff
@@ -328,6 +328,14 @@ public int numConnections() {
         return connectionLimitingHandler.numConnections();
     }
 
+    /**
+     * Waits until the result of {@link CompletableFuture} which is completed after the {@link #close()} or
+     * {@link #closeAsync()} operation is completed.
+     */
+    public void blockUntilShutdown() throws Exception {
```

Comment:
Could you use more specific `throws` clause? e.g. `throws InterruptedException, ...`